code/src/App.js

```diff
@@ -1,9 +1,21 @@
-import React from 'react'
+import React from "react";
+
+import { Form } from './Form'
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+	return (
+		<div className="App">
+			<div className="header-section" >
+				<h1 className="title" tabIndex="0">HÖRLÜRAR</h1>
+				<p className="title-line-one" tabIndex="0"> What is music to your ears !!</p>
+				<p className="title-line-two" tabIndex="0">Tell us more about your headphones...</p>
+			</div>
+			<div className="headphone-image" tabIndex="0">
+				<img src="./img/headphone.svg" alt="Headphone"></img>
+			</div>
+			<div>
+				<Form />
+			</div>
+		</div>
+	);
+};
```

Comment:
It looks like you are not consistent with your indentation. In some places, you have a lot of spacing, in other places you don't.